packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,48 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
```

Comment:
This file does not belong to `util` in its current form. This is a module with side effects (creates DOM) and not a utility such as `camelToKebabCase`.